components/gateway/README.md

```diff
@@ -8,36 +8,48 @@ Gateway is a component that exposes a service through which it forwards the inco
 
 Gateway binary allows you to override some configuration parameters. You can specify the following basic environment variables:
 
-| Name                             | Description                                                     | Default value            | 
+| Name                             | Description                                                       | Default value            | 
 | ---------------------------------| ----------------------------------------------------------------- | ------------------------ | 
 | **APP_ADDRESS**                  | The address and port for the service to listen on                 | `http://127.0.0.1:3000`  | 
 | **APP_DIRECTOR_ORIGIN**          | The address and port on which the Director service is listening   | `http://127.0.0.1:3001`  | 
 | **APP_CONNECTOR_ORIGIN**         | The address and port on which the Connector service is listening  | `http://127.0.0.1:3002`  | 
 | **APP_AUDITLOG_ENABLED**         | The variable that enables the audit log feature                   | `false`                  | 
 
-### Audit log authorization configuration
+### Audit log configuration
 
-If you set **APP_AUDITLOG_ENABLED** to `true`, the following environment variables are required:
+If you set **APP_AUDITLOG_ENABLED** to `true`, you must specify the following environment variables:
 
-| Name                             | Description                                                                     | 
-| -------------------------------- | ------------------------------------------------------------------------------- |
-| **APP_AUDITLOG_URL**             | The URL under which the audit log service is available                          |
-| **APP_AUDITLOG_CONFIG_PATH**     | The path for logging configuration changes                                      |
-| **APP_AUDITLOG_SECURITY_PATH**   | The path for logging security events                                            |
-| **APP_AUDITLOG_AUTH_MODE**       | The audit log authorization mode. The possible values are `basic` and `oauth`.  |
+| Name                             | Description                                                                       |   Default value  | 
+| -------------------------------- | --------------------------------------------------------------------------------- | ---------------- | 
+| **APP_AUDITLOG_URL**             | The URL under which the audit log service is available                            |      None        | 
+| **APP_AUDITLOG_CONFIG_PATH**     | The path for logging configuration changes                                        |      None        | 
+| **APP_AUDITLOG_SECURITY_PATH**   | The path for logging security events                                              |      None        | 
+| **APP_AUDITLOG_AUTH_MODE**       | The audit log authorization mode. The possible values are `basic` and `oauth`.    |      None        |  
```

Comment:
I guess there must be any default value here 🤔 What is the authorization method if the user does not specify any?

Reply:
The component won't start.